p2p/gen/openapi/validatorapi/v1/validatorapi.swagger.yaml

```diff
@@ -0,0 +1,102 @@
+swagger: "2.0"
+info:
+  title: Validator API
+  version: 1.0.0-alpha
+  license:
+    name: Business Source License 1.1
+    url: https://github.com/primev/mev-commit/blob/main/LICENSE
+consumes:
+  - application/json
+produces:
+  - application/json
+paths:
+  /v1/validator/get_epoch:
+    get:
+      summary: GetEpoch
+      description: GetEpoch is called by the bidder to get the current epoch, the current justified epoch, the previous justified epoch, and the finalized epoch.
+      operationId: Validator_GetEpoch
+      responses:
+        "200":
+          description: A successful response.
+          schema:
+            $ref: '#/definitions/v1EpochResponse'
+        default:
+          description: An unexpected error response.
+          schema:
+            $ref: '#/definitions/googlerpcStatus'
+  /v1/validator/get_validators:
+    get:
+      summary: GetValidators
```

Comment:
Is there a reason why we separate this out from `GetEpoch`? Ie. why not just have a single endpoint that returns the validators of the latest epoch? 

Reasoning here should prob be in our docs

Reply:
Asked Evan, he said that it's sufficient to `GetEpoch` be done automatically in `GetValidators` request